VERSION

```diff
@@ -1 +1 @@
-0.4.0
```

Comment:
I don't see a reason for this file anymore. Can we just remove it?

Reply:
Opened https://github.com/container-storage-interface/spec/issues/324 to track that.